Expected `u32` found `u64` compilation error on ARMv7 (or maybe all 32-bit arches?)

```
error[E0308]: mismatched types
   --> src/lib.rs:148:69
    |
148 |         match unsafe { ffi::fvad_process(self.fvad, frame.as_ptr(), frame.len() as u64) } {
    |                                                                     ^^^^^^^^^^^^^^^^^^ expected `u32`, found `u64`
    |
help: you can convert a `u64` to a `u32` and panic if the converted value doesn't fit
    |
148 |         match unsafe { ffi::fvad_process(self.fvad, frame.as_ptr(), (frame.len() as u64).try_into().unwrap()) } {
    |                                                                     +                  +++++++++++++++++++++

For more information about this error, try `rustc --explain E0308`.
```

## How to reproduce

Unless you have a capable ARMv7 thing lying around, it'll probably be more convenient to cross-compile to reproduce this.

Install [cross](https://github.com/cross-rs/cross):
```
cargo install cross --git https://github.com/cross-rs/cross
```

Clone the repo and cd into it:

```
git clone https://github.com/rvolosatovs/fvad && cd fvad
```

Create `armv7.Dockerfile` in the same directory:

```docker
FROM ghcr.io/cross-rs/armv7-unknown-linux-gnueabihf:main

RUN dpkg --add-architecture armhf && \
    apt-get update && \
    apt-get install --assume-yes libasound2-dev:armhf libssl-dev:armhf libc++-dev:armhf clang-3.9
```

Build:

```shell
docker build -t unpublished:armv7-unknown-linux-gnueabihf -f armv7.Dockerfile .
```

Create `Cross.toml` in the same directory:

```toml
[target.armv7-unknown-linux-gnueabihf]
image = "unpublished:armv7-unknown-linux-gnueabihf"
```

Build with Cross:
```
cross build --target armv7-unknown-linux-gnueabihf
```

---

I don't know how this is meant to be solved. There might be some bindgen trick that I'm not aware of (it seems like it should be typed at `usize`??) or maybe the source code just has to be 'coincidentally compatible' with both `u32` and `u64`...?

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -145,7 +145,7 @@ impl Fvad {
     /// - `Some(false)` on no active voice detection
     /// - `None` on invalid frame length
     pub fn is_voice_frame(&mut self, frame: &[i16]) -> Option<bool> {
-        match unsafe { ffi::fvad_process(self.fvad, frame.as_ptr(), frame.len() as u64) } {
+        match unsafe { ffi::fvad_process(self.fvad, frame.as_ptr(), frame.len()) } {
             -1 => None,
             0 => Some(false),
             1 => Some(true),

